pyproject.toml

```diff
@@ -18,39 +18,43 @@ classifiers = [
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Unix',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-numpy = "^1.19.5"
-pillow = "^8.3.2"
-smqtk-core = ">=0.18.0"
-smqtk-dataprovider = ">=0.16.0"
+python = "^3.8"
+pillow = ">=9.2.0"
+numpy = ">=1.22"
+smqtk-core = "*"
```

Comment:
I'm not sure my feelings on `"*"` usage have fully annealed, but I guess I'm okay with it as long as we are not aware of, or able to isolate, a minimum feature requirement. In this case, I think we want to base ourselves on `>=0.19` of smqtk-core as there is a behavior change in the Pluggable that removed some wonky `__new__` overriding.